tests/cmdline/params/options/test_callable.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+# pylint: disable=redefined-outer-name
+"""Tests for the :mod:`aiida.cmdline.params.options.callable` module."""
+from click.shell_completion import ShellComplete
+import pytest
+
+from aiida.cmdline.commands.cmd_verdi import verdi
+
+
+def _get_completions(cli, args, incomplete):
+    comp = ShellComplete(cli, {}, cli.name, '_CLICK_COMPLETE')
+    return comp.get_completions(args, incomplete)
+
+
+@pytest.fixture
+def unload_config():
+    """Temporarily unload the config by setting ``aiida.manage.configuration.CONFIG`` to ``None``."""
+    from aiida.manage import configuration
+
+    config = configuration.CONFIG
+    configuration.CONFIG = None
```

Comment:
NOTE: I have removed the try-except block, I don't think it is necessary, `pytest` should ensure that the fixture is run to completion after the test, unless the fixture itself excepts before the yield point, but here we only have two assignments.

https://docs.pytest.org/en/latest/how-to/fixtures.html#teardown-cleanup-aka-fixture-finalization

https://docs.pytest.org/en/latest/how-to/fixtures.html#safe-teardowns